src/controllers/secure.register.filter.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response, NextFunction } from "express";
+import * as config from "../config";
+import { logger } from "../utils/logger";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.SECURE_REGISTER_FILTER_PAGE}`);
+    return res.render(config.SECURE_REGISTER_FILTER_PAGE, {
+      backLinkUrl: config.SOLD_LAND_FILTER_URL
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.SECURE_REGISTER_FILTER_PAGE}`);
+    const isSecureRegister = req.body.secure_register;
+    if (isSecureRegister === '1') {
+      return res.render(config.USE_PAPER_PAGE, {
```

Comment:
line 22 - i think this will need to be a redirect to the paper page. If we use render, then the page url won't change and the matomo journey tracking won't pick up that they have essentially moved to a new page (we had to change the stop pages to redirects in confirmation-statement for matomo)

Reply:
Ah right, we'll have to do something similar for the other page I added too. Think it's possible to make a generic cannot use page? Otherwise it could result in a whole load of different controllers

Reply:
It's not a lot of code, it's probably more readable for anyone picking this up in the future to keep them separate.